CHANGELOG.rst

```diff
@@ -24,6 +24,8 @@ Changelog
 * Add Maestro credit card. Thanks @anthonylauzon.
 * Add ``hr_HR`` localization. Thanks @mislavcimpersak.
 * Update ``de_DE`` first names. Thanks @WarrenFaith and @mschoebel.
+* Allow generation of IPv4 and IPv6 network address with valid CIDR.
+* Unittest IPv4 and IPv6 address and network generation.
```

Comment:
Done in commit: https://github.com/kdeldycke/faker/commit/a9c021717cf2a45b09a9ffc00b8897cd1df31fbd